crates/core/src/auth/identity.rs

```diff
@@ -27,21 +29,17 @@ pub fn encode_token(private_key: &EncodingKey, identity: Identity) -> Result<Str
 pub fn encode_token_with_expiry(
     private_key: &EncodingKey,
     identity: Identity,
-    expiry: Option<u64>,
+    expiry: Option<Duration>,
 ) -> Result<String, JwtError> {
     let header = Header::new(jsonwebtoken::Algorithm::ES256);
 
-    let expiry = expiry.map(|seconds| {
-        let mut timer = SystemTime::now();
-        timer += Duration::from_secs(seconds);
-        // SAFETY: duration_since will panic if an argument is later than the time
-        // used for the duration calculation. In case of UNIX_EPOCH it can't be the case
-        timer.duration_since(UNIX_EPOCH).unwrap().as_secs()
```

Comment:
why did this get to be removed?

Reply:
Tidying up auth stuff, I guess. serde_with can do this conversion for us.

Reply:
Ah I see the new `serde_with` line. Seems reasonable.